odoo/osv/expression.py

```diff
@@ -143,7 +144,7 @@
 # operators are also used. In this case its right operand has the form (subselect, params).
 TERM_OPERATORS = ('=', '!=', '<=', '<', '>', '>=', '=?', '=like', '=ilike',
                   'like', 'not like', 'ilike', 'not ilike', 'in', 'not in',
-                  'child_of', 'parent_of')
+                  'child_of', 'parent_of', 'any', 'not any')
```

Comment:
Are those operators public?
I would expect these to be internal operators like `inselect`.

Reply:
We wanted to make them internal but we cannot as `any` operators can be passed to `_search` methods and themselves back inside of expression.py.

Reply:
Best we can do is to leave them undocumented for now

Reply:
Making them public is not an issue, and it enables to write down some domains that simply _cannot_ be written with existing operators, for instance:
```py
# all sales orders that sell at least 3 units of a product with id 42
[('order_line', 'any', [('product_id', '=', 42), ('product_uom_qty', '>=', 3)])]
```
The reason `inselect` cannot be public is that it would enable SQL injections.  In contrast, operators `any` and `not any` are safe by construction.